### ISSUE TITLE
Family history of pulmonary hypertension

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6554]
**Preferred term label:** Family history of pulmonary hypertension

**Synonyms**

**Definition (free text, please give PubMed ID)**
Family history of pulmonary hypertension with or without mutation in known risk genes

**Parent term (use hpo.jax.org/app)**
Family history HP:0032316

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
Please use separate field to record family history, with Mondo term for instance.
